cgi-bin/DW/External/Site.pm

```diff
@@ -96,7 +130,8 @@ sub get_site {
                 map { lc $_ }
                 split( /\./, $site );
 
-    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
+    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
```

Comment:
Yes! Sorry I wasn't clear I meant we can cut out a lot of the complexity of the patch by removing the changes in the individual DW::External:::Site classes :)

Also! I forgot to say this earlier but please include the issue number in your commit message (so we can track which one it was) 




> On May 10, 2015, at 04:21, kaberett <notifications@github.com> wrote:
> 
> In cgi-bin/DW/External/Site.pm:
> 
> > @@ -96,7 +130,8 @@ sub get_site {
> >                  map { lc $_ }
> >                  split( /\./, $site );
> >  
> > -    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
> > +    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
> So those trailing conditions pre-exist my touching the code, sooo I'm not entirely sure what they're for. Happy to remove them if you're confident we don't need them but I assumed they wanted leaving well enough alone because they'd been put there for a reason! :-)
> 
> —
> Reply to this email directly or view it on GitHub.
>